### PR TITLE
Shows most recent run info on homepage (if there is some)

### PR DESCRIPTION
**Pull request by elsigh**
The created date for a build isn't super interesting info, but the last run and the status is.

### CODE REVIEW

File: dpxdt/server/frontend.py [hunk 1]
```diff
@@ -56,9 +56,27 @@ def homepage():
         auth.claim_invitations(current_user)
 
     build_list = operations.UserOps(current_user.get_id()).get_builds()
+
+    # Builds a dict of the most recent releases for each build that has
+    # a relaease.
+    releases = {}
+    for build in build_list:
+        logging.info('BUILD: %s' % build)
+        last_release = (
```
Comment (bslatkin) [inline dpxdt/server/frontend.py]: Can you move this query into operations.py? I'm trying to do everything like this in there so we can have a clear boundary for caching.

I also think we can convince the ORM to do this look-up for you as a join on the first look-up in get_builds.

We'll need to add something like:

```python
last_release = db.relationship(
    'Release',
    lazy='joined',
    join_depth=1,
    order_by='Release.created.desc')
```

And then you can just access `build.last_release` in the template and you'll have a whole `Release` ORM object there for rendering. This is a bit finicky but let me know if you need help. The ORM docs to read are here:

<http://docs.sqlalchemy.org/en/improve_toc/orm/relationship_api.html>

Comment (elsigh) [inline dpxdt/server/frontend.py]: That was really easy!

**Comment by bslatkin** [comment]
Looks like Travis is unhappy with it because of the ORM. Let me try to patch this tomorrow and see if I can reproduce. My guess is the release is getting loaded twice and we need to make the attribute lazy.

**Comment by elsigh** [comment]
Cool thanks. It seems with each cl I write I make work for you too. :)
Hopefully I'll get more self sufficient.

**Pull request closed without merging**